src/raster/r.exdet/r.exdet.html

```diff
@@ -100,7 +100,7 @@ <h2>CITATION</h2>
 
 <ul><li>van Breugel, P. (2016) r.exdet, a GRASS GIS addon for the
 quantification of novel uni- and multi-variate environments. URL:
-https://grass.osgeo.org/grass70/manuals/addons/r.exdet.html</li></ul>
+https://grass.osgeo.org/grass7/manuals/addons/r.exdet.html</li></ul>
```

Comment:
This still redirects to grass78, is that what is wanted? 
In the #664 PR I let it alone initially since it was a citation and thought that the reference was valid at that time.

Reply:
ok, restored in f6ac6f802258fb70cbd4fe35eedb3e833e9c7020